sirepo/template/srw.py

```diff
@@ -1286,6 +1255,52 @@ def _enum_text(name, model, field):
     return ''
 
 
+def _export_rsopt_config(data):
+    v = _rsopt_jinja_context(data.models.exportRsOpt)
+    filename = 'rsOptExport.zip'
+    f = re.sub(r'[^\w.]+', '-', pkio.py_path(filename).purebasename).strip('-')
```

Comment:
I think although don't know, that `.` should be `\.`, since dot means "anything" so the regex does nothing afaict. What is f supposed to be?

Reply:
moot due to [075b377](https://github.com/radiasoft/sirepo/pull/4057/commits/075b377ec4e22e26dac2d429ee1d02791ddfeaf1)